include/yaramod/types/tokenstream.h

```diff
@@ -0,0 +1,135 @@
+#pragma once
```

Comment:
I would be for renaming this file to `token_stream.h` because we refer to it as `TokenStream` so there should be some spacing between token and stream.

Reply:
Done